diversity-scholarships/angel-fund.html

```diff
@@ -19,16 +18,15 @@
 					page, though anonymous donation is also possible.
 				</p>
 				<p>
-					<a class="btn btn-primary btn-lg" href="https://mxcustomer.com/d/bff4609a-b02f-48e6-abbd-5784a7140e49">Donate Now</a>
+					<a class="btn btn-primary btn-lg" href="https://mxmerchant.com/mxcustomer/d/4c19513a-51d3-4f1a-972a-06fd0eefb880">Donate Now</a>
 				</p>
 				<p>
 					All funds raised will help pay for transportation, accommodation, and conference registration costs
 					for Code4Lib's Diversity Scholarships recipients for our annual conference. Please feel free to
-					direct any question to the <a href="http://wiki.code4lib.org/2016_Conference_Committees#Scholarship_Committee">Scholarship Committee</a>
+					direct any question to the <a href="https://wiki.code4lib.org/2017_Conference_Committees#Scholarship_Committee">Scholarship Committee</a>
 					via chadbnelson at gmail dot com.
 				</p>
```

Comment:
need to update this email.  I just emailed Linda asking if it was ok to place her email here.